storage/innobase/xtrabackup/src/changed_page_tracking.cc

```diff
@@ -189,11 +189,11 @@ bool get_changed_pages(lsn_t start_lsn, lsn_t end_lsn, uint64_t *backupid,
   }
 
   /* validate tracking lsn */
-  lsn_t backup_start_lsn = get_pagetracking_start_lsn(connection);
+  lsn_t page_track_start_lsn = get_pagetracking_start_lsn(connection);
 
-  if (backup_start_lsn == 0) {
+  if (page_track_start_lsn == 0) {
     return (false);
-  } else if (backup_start_lsn > start_lsn) {
+  } else if (page_track_start_lsn > start_lsn) {
     msg("xtrabackup: pagetracking: Warning tracking lsn " LSN_PF
         " is more than increment start lsn " LSN_PF "\n",
         backup_start_lsn, start_lsn);
```

Comment:
did this compile? :) because backup_start_lsn is removed but we use it for printing here? :)

Reply:
Fixed it.